tests/manage/storageclass/test_create_multiple_sc_with_different_pool_name.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+import logging
+from tests import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs import defaults
+from ocs_ci.ocs.resources.pod import get_fio_rw_iops
+from ocs_ci.ocs.exceptions import ResourceLeftoversException
+from ocs_ci.framework.testlib import ManageTest, tier1
+from tests.fixtures import (
+    create_rbd_secret, create_project
+)
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def resources(request):
```

Comment:
the same fixture exists under manage/cluster/nodes
I suggest 2 options:
- Move the fixtures from nodes dir to tests/conftest.py and call it from here and from nodes related tests
- Use the new factory fixtures from conftest instead of this one

Reply:
Done